conjur/controller/init_controller.py

```diff
@@ -124,7 +126,7 @@ def get_server_certificate(self, conjur_url: ParseResult) -> Optional[str]:
                          "openssl x509 -fingerprint -noout -in ~conjur/etc/ssl/conjur.pem\n\n")
         trust_certificate = input("Trust this certificate? yes/no (Default: no): ").strip()
         if trust_certificate.lower() not in VALID_CONFIRMATIONS:
-            raise RuntimeError("You decided not to trust the certificate")
+            raise CertificateNotTrust("You decided not to trust the certificate")
```

Comment:
Should be finish with error or exception

Reply:
@sigalsax  Why this is an exception in the first place?
I would prompt a message and return. If you agree than @royico please leave a TODO that we should change this

Reply:
It is an error because the user _failed_ to trust the certificate. In other words, he performed an action that warrants an error. If we just return, it is like returning error code 0 which isn't accurate because in no way was the user successful.